easybuild/easyconfigs/p/pygmo/pygmo-2.19.5-foss-2022b.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'CMakePythonPackage'
+
+name = 'pygmo'
+version = '2.19.5'
+
+homepage = 'https://esa.github.io/pygmo2'
+description = "pygmo is a scientific Python library for massively parallel optimization."
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = ['https://github.com/esa/pygmo2/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['742ff0c3d535cb2af94d5095968d8f29c23ed7bdbd4ddd8259ca787eef881aa2']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('pybind11', '2.10.3'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('Boost', '1.81.0'),
+    ('pagmo', '2.19.0'),
+    ('matplotlib', '3.7.0'),
+    ('networkx', '3.0'),
```

Comment:
ah, unfortunate, this was networkx-3.0 was added at foss level, should have been gfbf, guess we'll just keep that in mind and address it in 2023a going foward (just ignore this comment, this stays at foss level as a result of this)

Reply:
I also like going with the lowest possible toolchains so would it really cause any problems if networkx-3.0 was dropped to the gfbf toolchain and the foss variant removed?  It's only used in two packages and those who have already installed it won't be affected and if they need it as a new gfbf dependency in a new EB, it will just install then.  Anyone who has never installed it will just go straight with the gfbf version.